### PR TITLE
Sort imports alphabetically and remove/add some spaces

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: paper-input-container.html [hunk 1]
```diff
@@ -177,9 +176,10 @@
       }
 
       .input-content {
-        position: relative;
         @apply(--layout-horizontal);
         @apply(--layout-center);
+
+        position: relative;
```
Comment (notwaldorf) [inline paper-input-container.html]: why is this change needed?

File: paper-input-container.html [hunk 2]
```diff
@@ -198,18 +198,17 @@
 
       .input-content.label-is-floating ::content label,
       .input-content.label-is-floating ::content .paper-input-label {
+        /* Since we scale to 75/100 of the size, we actually have 100/75 of the
+        original space now available */
+        width: 133%;
```
Comment (notwaldorf) [inline paper-input-container.html]: why is this change needed?

File: paper-input-container.html (hunk 1 above)
Comment (abdonrd) [inline paper-input-container.html]: It is not necessary, but most of the elements have the external `@apply()`'s at the beginning.

I think it would be awesome to have a style guide of styles for the elements. :smiley:

File: paper-input-container.html (hunk 2 above)
Comment (abdonrd) [inline paper-input-container.html]: I dont't know why this change. :confused:

Back!

File: paper-input-container.html (hunk 1 above)
Comment (notwaldorf) [inline paper-input-container.html]: Yeah a style guide would be neat. I think this particular change is ok, but moving CSS around just for the sake of moving it makes me a bit worried, since sometimes before or after a mixin has a meaning (`--layout-fit`, for example, applies `position:relative`, so in some cases if you want to override that, before or after the mixin is applied is actually important)

Comment (abdonrd) [inline paper-input-container.html]: Yes, you're absolutely right! **Cascading** Style Sheets... :grin:
